RELEASE_NOTES.txt

```diff
@@ -1,6 +1,141 @@
 Apache Ignite Release Notes
 ===========================
 
+Apache Ignite In-Memory Distributed Database 2.12.0
+-----------------------------------------------------------
+
+(!) WARNINGS:
+* The community accepted deprecate for removal in future releases: CacheMode#LOCAL, CacheAtomicityMode#TRANSACTIONAL_SNAPSHOT, CacheConfiguration#rebalanceDelay.
```

Comment:
We should mention that legacy service grid implementation is already removed in the next relese.

Reply:
Fixed.